pennylane/ops/qubit.py

```diff
@@ -381,6 +418,15 @@ class Rot(Operation):
     par_domain = "R"
     grad_method = "A"
 
+    @staticmethod
+    def decomposition(phi, theta, omega, wires=None):
+        decomp_ops = [
+            RZ(phi, wires=wires),
+            RY(theta, wires=wires),
+            RZ(omega, wires=wires)
+        ]
+        return decomp_ops
```

Comment:
Why don't we just go for
```python
return [
    RZ(phi, wires=wires),
    RY(theta, wires=wires),
    RZ(omega, wires=wires),
]
```
Having a separate variable for this seems superfluous.

Reply:
Actually I already found a use case for forcing a decomposition here. I can't construct a metric tensor for `Rot` gates, but if they're decomposed I can.